bin/install_pywps.sh

```diff
@@ -178,6 +178,9 @@ echo 'creating Apache configuration'
 
 cat << EOF > "$PYWPS_APACHE_CONF"
 WSGIScriptAlias /pywps/wps.py $PYWPS_WSGI
+<Directory "$PYWPS_WSGI">
```

Comment:
That should be: `<Directory "$PYWPS_APP">` to use the parent directory of the WSGI script instead of the script itself.